instrument-starters/opentracing-spring-cloud-aop/src/main/java/io/opentracing/contrib/spring/cloud/aop/Traced.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2017-2018 The OpenTracing Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.opentracing.contrib.spring.cloud.aop;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target({ElementType.METHOD, ElementType.ANNOTATION_TYPE, ElementType.TYPE})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Traced {
+
+  boolean withActiveSpanOnly() default true;
+
+  String component() default "traced";
+
+  String operationName() default "";
```

Comment:
What is the default value? FQCN with method name?

Reply:
Add also value as alias for this

Reply:
value as alias 👍 
As for the default value it's the method name as per: https://github.com/opentracing-contrib/java-spring-cloud/blob/b16095406387658e4914b79accf407f886a9d421/instrument-starters/opentracing-spring-cloud-core/src/main/java/io/opentracing/contrib/spring/cloud/aop/BaseTracingAspect.java#L89-L91

 
We also capture them as tag here https://github.com/opentracing-contrib/java-spring-cloud/blob/b16095406387658e4914b79accf407f886a9d421/instrument-starters/opentracing-spring-cloud-core/src/main/java/io/opentracing/contrib/spring/cloud/aop/MethodInterceptorSpanDecorator.java#L72-L75